test/customer_test.rb

```diff
@@ -1,8 +1,8 @@
-require 'minitest/autorun'
-require 'minitest/reporters'
-require 'minitest/skip_dsl'
+require "minitest/autorun"
+require "minitest/reporters"
+require "minitest/skip_dsl"
 
-require_relative '../lib/customer'
+require_relative ../lib/customer"
```

Comment:
Syntax error.  Missing `"`

```suggestion
require_relative "../lib/customer"
```